git-remote-mediawiki

```diff
@@ -963,7 +970,12 @@ sub mw_import_revids {
 		$commit{comment} = $rev->{comment} || EMPTY_MESSAGE;
 		$commit{title} = smudge_filename($page_title);
 		$commit{mw_revision} = $rev->{revid};
-		$commit{content} = mediawiki_smudge($rev->{'*'});
+		if ( defined $rev->{'*'} ) {
+			$commit{content} = mediawiki_smudge($rev->{'*'});
+		} else {
+			# What do we do with suppressed revisions?
```

Comment:
It's not clear why this comment is here. Isn't the current version satisfactory with suppressed revisions? If not, what could be improved (document this in the comment)?

Reply:
thanks, please check the latest.